Description: avoid not-in-Debian crate thirtyfour_sync
Author: Jonas Smedegaard <dr@jones.dk>
Forwarded: not-needed
Last-Update: 2023-08-24
---
This patch header follows DEP-3: http://dep.debian.net/deps/dep3/
--- a/Cargo.toml
+++ b/Cargo.toml
@@ -122,7 +122,6 @@
 predicates = "3.1.0"
 serial_test = ">= 2.0.0, <= 3"
 test-case = "3.3.1"
-thirtyfour_sync = "0.27"
 
 [target.'cfg(unix)'.dependencies]
 daemonize = "0.5"
--- a/tests/oauth.rs
+++ b/tests/oauth.rs
@@ -1,5 +1,5 @@
 #![deny(rust_2018_idioms)]
-#![cfg(feature = "dist-client")]
+#![cfg(all(feature = "dist-client", feature = "skip"))]
 
 use fs_err as fs;
 use std::io::{self, Read, Write};
